eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -147,11 +147,11 @@ jobs:
       maxParallel: $[ variables['MaxParallelTestJobs'] ]
       matrix:
         Linux:
-          Pool: azsdk-pool-mms-ubuntu-1804-general
+          Pool: azsdk-pool-mms-ubuntu-1804-experimental
           OSVmImage: MMSUbunut18.04
           TestTargetFramework: netcoreapp2.1
         Windows_NetCoreApp:
-          Pool: azsdk-pool-mms-win-2019-general # Comment out to swap back to public hosted pool.
```

Comment:
nit: might want to remove or change these comments now.

Reply:
Yeah, I'll do some further clean-up here. This PR will never get merged though, its for testing purpsoes.